### ISSUE TITLE
Enhancement - bybit fetch_order()

### ISSUE THREAD

**Issue by p2828** [ccxt/ccxt#11839]
- OS: win7
- Programming Language version: Python 3.7.1
- CCXT version: 1.72.15

Issue:
I recently found that fetch_order() return had a key called 'price' that gave a applicable but wrong value. But that is because of the bybit. So, i had a small chat with the bybit support. 

Chat with bybit support:
________________________________________
me:
'btc/usdt' - testnet 
1. by web-UI:  
    current price = 37552.50 
2. market buy placed for qty = 0.01 
3. by /private/linear/order/search:  
    'last_exec_price': '37511.5' (ok) 
    'price': : '39387' (what !??) 
'price' value is 4.89% higher that current price 
what is 'price'? the documentation says,  
price = 'Order price'  
but then shouldn't it be near to the current price? 
but it is a whopping 4.89% higher than current 
price (for a 'buy' order). what's going on? 
ref: https://bybit-exchange.github.io/docs/linear/#t-queryactive

bybit:
Has the order been executed? What entry price do you see on testnet.bybit.com?

me:
Order is executed. entry price on testnet.bybit.com is '37511.5'

bybit:
A traditional market price order which will be filled at the best available price. "price" is not required for this type of order. (It is for limit orders). So the price that was executed is last_exec_price.

me:
so, as i understand, 'price' value can (or rather should) be ignored for market order. is it?

bybit:
Yes

me:
tnx
________________________________________

So, basically, if the 'price' can have a None or just a blank field in case of market orders, that would be convenient and less confusing.
Of course its not a very big issue for me, but just in case you want to take interest in.
And true, its bybit that needs to correct that but it would not be a bad thing to do it in ccxt too. 
You decide to do it or not, in either case, tnx buddy.

**Comment by p2828** [comment]
create_order with price set as 'undefined'
```
# input:
order = bybit_exchange.create_order(
    symbol = symbol, 
    order_type = 'market', 
    side = side, 
    amount = amount, 
    price = 'undefined', # or 0.00 - both work the same
    {
        'stop_loss': 30000,
        'take_profit': 40000.0,
    }
    )
order = pprint.pformat(order)
print(order)
with open('delme_order.txt', 'w') as f:
    f = f.write(str(order))

# output:
{'amount': 0.01,
 'average': None,
 'clientOrderId': None,
 'cost': 0.0,
 'datetime': None,
 'fee': {'cost': 0.0, 'currency': 'USDT'},
 'fees': [{'cost': 0.0, 'currency': 'USDT'}],
 'filled': 0.0,
 'id': '0986c329-c001-49a4-a9c0-03fe4ced3516',
 'info': {'close_on_trigger': False,
          'created_time': '2022-02-04T12:37:38Z',
          'cum_exec_fee': '0',
          'cum_exec_qty': '0',
          'cum_exec_value': '0',
          'last_exec_price': '0',
          'order_id': '0986c329-c001-49a4-a9c0-03fe4ced3516',
          'order_link_id': '',
          'order_status': 'Created',
          'order_type': 'Market',
          'position_idx': '1',
          'price': '39885.5',   # ==> note this gibrish value
          'qty': '0.01',
          'reduce_only': False,
          'side': 'Buy',
          'sl_trigger_by': 'LastPrice',
          'stop_loss': '30000',
          'symbol': 'BTCUSDT',
          'take_profit': '40000',
          'time_in_force': 'ImmediateOrCancel',
          'tp_trigger_by': 'LastPrice',
          'updated_time': '2022-02-04T12:37:38Z',
          'user_id': '246274'},
 'lastTradeTimestamp': None,
 'postOnly': False,
 'price': 39885.5,              # ==> note this gibrish value
 'remaining': 0.01,
 'side': 'buy',
 'status': 'open',
 'stopPrice': None,
 'symbol': 'BTC/USDT',
 'timeInForce': 'IOC',
 'timestamp': None,
 'trades': [],
 'type': 'market'}
```


fetch_order result after a few seconds
```
# input:
order = bybit_exchange.fetch_order( '0986c329-c001-49a4-a9c0-03fe4ced3516', symbol )
    all_orders = pprint.pformat(all_orders)
    print (all_orders)
    with open('delme_fetch_order.txt', 'w') as f:
        f = f.write(str(all_orders))

# output:
{'amount': 0.01,
 'average': 37986.5,
 'clientOrderId': None,
 'cost': 379.865,
 'datetime': None,
 'fee': {'cost': 0.28489875, 'currency': 'USDT'},
 'fees': [{'cost': 0.28489875, 'currency': 'USDT'}],
 'filled': 0.01,
 'id': '0986c329-c001-49a4-a9c0-03fe4ced3516',
 'info': {'close_on_trigger': False,
          'created_time': '2022-02-04T12:37:38Z',
          'cum_exec_fee': '0.28489875',
          'cum_exec_qty': '0.01',
          'cum_exec_value': '379.865',
          'last_exec_price': '37986.5',
          'order_id': '0986c329-c001-49a4-a9c0-03fe4ced3516',
          'order_link_id': '',
          'order_status': 'Filled',
          'order_type': 'Market',
          'price': '39885.5',           # ==> note this gibrish value
          'qty': '0.01',
          'reduce_only': False,
          'side': 'Buy',
          'sl_trigger_by': 'LastPrice',
          'stop_loss': '30000',
          'symbol': 'BTCUSDT',
          'take_profit': '40000',
          'time_in_force': 'ImmediateOrCancel',
          'tp_trigger_by': 'LastPrice',
          'updated_time': '2022-02-04T12:37:38Z',
          'user_id': '246274'},
 'lastTradeTimestamp': None,
 'postOnly': False,
 'price': 39885.5,                      # ==> note this gibrish value
 'remaining': 0.0,
 'side': 'buy',
 'status': 'closed',
 'stopPrice': None,
 'symbol': 'BTC/USDT',
 'timeInForce': 'IOC',
 'timestamp': None,
 'trades': [],
 'type': 'market'}
```
